2015/presentations/import_deep_dive/text.md

```diff
@@ -0,0 +1,297 @@
+# Import Deep Dive Detours - Petr Viktorin
+
+My talk, Import Deep Dive, gave you a simplified overview of Python's import
+machinery, enough to understand what's going on and where to look for more
+details.
```

Comment:
Ah. I understood the guidelines wrong: where they say the article "can be a supplement of the topic", I thought I can skip the main topic altogether. Perhaps that can be clarified for next year?

I've summarized my talk in the introduction now.